src/common/src/telemetry/mod.rs

```diff
@@ -29,8 +29,48 @@ use crate::util::resource_util::memory::{system_memory_available_bytes, total_me
 use crate::RW_VERSION;
 
 pub const TELEMETRY_CLUSTER_TYPE: &str = "RW_TELEMETRY_TYPE";
-const TELEMETRY_CLUSTER_TYPE_HOSTED: &str = "hosted"; // hosted on RisingWave Cloud
-const TELEMETRY_CLUSTER_TYPE_TEST: &str = "test"; // test environment, eg. CI & Risedev
+pub const TELEMETRY_CLUSTER_TYPE_HOSTED: &str = "hosted"; // hosted on RisingWave Cloud
+pub const TELEMETRY_CLUSTER_TYPE_TEST: &str = "test"; // test environment, eg. CI & Risedev
+pub const TELEMETRY_CLUSTER_TYPE_KUBERNETES: &str = "kubernetes";
+pub const TELEMETRY_CLUSTER_TYPE_SINGLE_NODE: &str = "single-node";
+pub const TELEMETRY_CLUSTER_TYPE_DOCKER_COMPOSE: &str = "docker-compose";
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum TelemetryClusterType {
+    Hosted,
+    Test,
```

Comment:
Was not updated along with the proto. 

Why not just use the `enum TelemetryClusterType` in the generated code?

Reply:
Unfortunately, the 'test' have been introduced along with scarf pr. So keep for compatibility.

Reply:
Does a breaking change of test affect us internal pipelines only or also users?

Reply:
Oh, the test pipeline does not enable telemetry, will remove later.